kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateListRow.js

```diff
@@ -45,8 +45,12 @@ define([
 
         function _updateActionButton(jobStateObject) {
             const jobAction = Jobs.jobAction(jobStateObject);
-            const jsActionString = jobAction.replace(/ /g, '-');
             const actionNode = container.querySelector('[data-element="job-action"] button');
+            if (!jobAction) {
+                actionNode.remove();
+                return;
+            }
```

Comment:
if the job ID is unknown, don't show any action (retry / cancel / go to results)